scripts/unit_test.sh

```diff
@@ -24,28 +28,31 @@ DATABASE_USER="compliance"
 DATABASE_PASSWORD="changeme"
 DATABASE_NAME="compliance-test"
 
+teardown() {
+
+  for id in "$DB_CONTAINER_ID" "$TEST_CONTAINER_ID"; do
+    cicd::container::cmd rm -f "$id" || echo "couldn't delete container ID: $id"
+  done
 
-function teardown_podman {
-  podman rm -f "$DB_CONTAINER_ID" || true
-  podman rm -f "$TEST_CONTAINER_ID" || true
-  podman pod rm -f "$COMPLIANCE_POD_ID" || true
+  cicd::container::cmd pod rm -f "$COMPLIANCE_POD_ID" || echo "couldn't delete pod ID: $COMPLIANCE_POD_ID"
 }
 
-trap "teardown_podman" EXIT SIGINT SIGTERM
+trap "teardown" EXIT SIGINT SIGTERM
 
-if ! COMPLIANCE_POD_ID=$(podman pod create --name "$POD_NAME"); then
+if ! COMPLIANCE_POD_ID=$(cicd::container::cmd pod create --name "$POD_NAME"); then
     exit 1
```

Comment:
@Victoremepunto, there is no information provided for this `exit 1`, maybe some hint could be added:

```suggestion
    echo "Failed creating pod image"
    exit 1
```

Reply:
That's useful 😁

Reply:
It wasn't a joke